main.md

```diff
@@ -117,6 +117,9 @@ NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED",
 described in BCP 14 [@!RFC2119] [@!RFC8174] when, and only when, they
 appear in all capitals, as shown here.
 
+This specification uses the Augmented Backus-Naur Form (ABNF) notation
+of [@!RFC5234], extended with the notation for case-sensitivity
+in strings defined in [RFC7405].
```

Comment:
```suggestion
This specification uses the Augmented Backus-Naur Form (ABNF) notation
of [@!RFC5234].
```

Good catch! If we were to repeat `token68` I think we need to import all the associated rules though (e.g DIGIT, ALPHA).

I noted that in this specification, the syntax of `credentials` is different from the one in 7235, that is 

```
     credentials = auth-scheme [ 1*SP ( token68 / #auth-param ) ]
```

moreover 7235 states

> The "token68" notation was introduced for compatibility with
      existing authentication schemes and can only be used once per
      challenge or credential.  Thus, new schemes ought to use the
      auth-param syntax instead, because otherwise future extensions
      will be impossible.